README.md

```diff
@@ -1,88 +1,60 @@
+To do this, set the target (replace `<TARGET_NAME>` with your target name) value `restrict_size` in mbed_app.json:
 
 ```
+...    
     "target_overrides": {
-        ...
         "<TARGET_NAME>": {
             "target.restrict_size": "0x20000"
-        },
-        ...
+        }
+    }
+...
 ```
 
-Note - `restrict_size` pads the build image to the requested size.
-
-## Now compile
-
-Invoke `mbed compile`, and specify the name of your platform and your favorite toolchain (`GCC_ARM`, `ARM`, `IAR`). For example, for the ARM Compiler 5:
-
-```
-mbed compile -m <TARGET_NAME> -t ARM
-```
-
-Your PC may take a few minutes to compile your code. At the end, you see the following result:
-
 ```
+mbed compile -m <TARGET_NAME> -t <toolchain>
+...
 Merging Regions:
   Filling region application with .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader_application.bin
   Padding region application with 0x11420 bytes
 Space used after regions merged: 0x20000
-+-----------------------------+-------+-------+-------+
-| Module                      | .text | .data |  .bss |
-+-----------------------------+-------+-------+-------+
-| Misc                        | 18481 |    24 |  1268 |
-| drivers                     |  2132 |     8 |   184 |
-| features/FEATURE_LWIP       |    46 |     0 | 12560 |
-| features/filesystem         | 12756 |    12 |   540 |
-| hal                         |   414 |     8 |     0 |
-| hal/TARGET_FLASH_CMSIS_ALGO |   520 |    28 |     0 |
-| platform                    |  2759 |    32 |   380 |
-| rtos                        |   144 |     8 |     0 |
-| rtos/rtx                    |  6954 |   100 |  8396 |
-| targets/TARGET_STM          | 15560 |   568 |   736 |
-| Subtotals                   | 59766 |   788 | 24064 |
-+-----------------------------+-------+-------+-------+
-Allocated Heap: unknown
-Allocated Stack: unknown
-Total Static RAM memory (data + bss): 24852 bytes
-Total RAM memory (data + bss + heap + stack): 24852 bytes
-Total Flash memory (text + data + misc): 60554 bytes
-
+...
 Image: .\BUILD\<TARGET_NAME>\ARM\mbed-os-example-bootloader.bin
 ```
 
-It creates two binary files. The original uncombined image is in the output directory, <project-name>_application.bin, and the bootloader image is <project-name>.bin.
+This creates two binary files. The original unmodified image is in the output directory, <project-name>_application.bin, and the padded bootloader image is <project-name>.bin.
 
-When the build succeeds, you have created a bootloader for your target. This bootloader defines the update file to be located at `/sd/mbed-os-example-bootloader-blinky_application.bin`. This is the binary bootloader flashes and then jumps to.
+Copy the <project-name>.bin image to your board and without an SD card present or application file loaded on the SD card you should see an output similar to:
+```
+???What does this example print out???
```

Comment:
This bootloader example doesn't verify the application before starting it, so if the application is missing the device will crash and repeatedly print out "No update found to apply" and "Starting application".

If there are directions to load just the bootloader (and not the composite image) then the code should probably be updated to check for a missing application.